pkg/kubelet/dockershim/docker_service.go

```diff
@@ -104,5 +115,10 @@ func (ds *dockerService) Version(_ string) (*runtimeApi.VersionResponse, error)
 }
 
 func (ds *dockerService) UpdateRuntimeConfig(runtimeConfig *runtimeApi.RuntimeConfig) error {
+	if ds.networkPlugin != nil && runtimeConfig.NetworkConfig.PodCidr != nil {
+		details := make(map[string]interface{})
+		details[network.NET_PLUGIN_EVENT_POD_CIDR_CHANGE_DETAIL_CIDR] = *runtimeConfig.NetworkConfig.PodCidr
```

Comment:
I think current network plugins don't persist PodCidr for us, maybe write it to a local file (e.g. `/var/lib/dockershim/config`)?